zeebe/docker/utils/startup.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh -xeu
 
 HOST=$(hostname -i)
+ZEEBE_SET_SEED_ENV_VARS=${ZEEBE_SET_SEED_ENV_VARS:-true}
 
 if [ "$ZEEBE_STANDALONE_GATEWAY" = "true" ]; then
-    export ZEEBE_GATEWAY_NETWORK_HOST=${ZEEBE_GATEWAY_NETWORK_HOST:-${HOST}}
+    if [ "$ZEEBE_SET_SEED_ENV_VARS" = "true" ]; then
+        export ZEEBE_GATEWAY_NETWORK_HOST=${ZEEBE_GATEWAY_NETWORK_HOST:-${HOST}}
+    fi
     export ZEEBE_GATEWAY_CLUSTER_HOST=${ZEEBE_GATEWAY_CLUSTER_HOST:-${ZEEBE_GATEWAY_NETWORK_HOST}}
```

Comment:
Probably we need it for the env var `ZEEBE_GATEWAY_CLUSTER_HOST` too.

Reply:
On a working instance, when I echo the value of `ZEEBE_GATEWAY_CLUSTER_HOST`, I see that it is the same as the output of `hostname -i`
So I don't think it has to be `0.0.0.0` on a helm installation.